src/test/java/models/testGame.java

```diff
@@ -19,7 +19,12 @@ public void testGameCreation(){
     public void testGameBuildDeck(){
         Game g = new Game();
         g.buildDeck();
-        assertEquals(52,g.deck.size());
+        if(g.mode == 1){
+            assertEquals(52,g.deck.size());
+        }
+        if(g.mode == 2){
+            assertEquals(40,g.deck.size());
+        }
     }
 
```

Comment:
We're probably going to have to have more tests than this to bring up test coverage up to snuff.